.ci/riscv-tests.sh

```diff
@@ -7,20 +7,23 @@ check_platform
 mkdir -p toolchain
 
 # GNU Toolchain for RISC-V
-GCC_VER=12.2.0-1
-XPACK_REPO=https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download
+GCC_VER=2023.01.04-nightly
+TOOLCHAIN_REPO=https://github.com/riscv-collab/riscv-gnu-toolchain/releases
+
+# Install RISCOF
+pip3 install git+https://github.com/riscv/riscof.git
 
 set -x
 
 wget -q \
-    ${XPACK_REPO}/v${GCC_VER}/xpack-riscv-none-elf-gcc-${GCC_VER}-linux-x64.tar.gz -O- \
+    ${TOOLCHAIN_REPO}/download/2023.01.04/riscv32-elf-ubuntu-20.04-nightly-${GCC_VER}.tar.gz -O- \
```

Comment:
Don't hard code `2023.01.04` in URL. Instead, specify via shell commands.

Reply:
We are using Ubuntu Linux 22.04 in CI pipeline.

Reply:
I will rewrite as following:
```
GCC_VER=2023.01.04

wget -q \
    ${TOOLCHAIN_REPO}/download/${GCC_VER}/riscv32-elf-ubuntu-20.04-nightly-${GCC_VER}-nightly.tar.gz -O- \
| tar -C toolchain -xz
```

Reply:
It looks fine. Specify `ubuntu-22.04` instead.